FaunaDB.Client/Properties/AssemblyInfo.cs

```diff
@@ -1,39 +1,5 @@
-﻿using System.Reflection;
-using System.Runtime.CompilerServices;
-
-// Information about this assembly is defined by the following attributes.
-// Change them to the values specific to your project.
-
-[assembly: AssemblyTitle("FaunaDB.Client")]
```

Comment:
I take it all this is replaced by just the project files now?

Reply:
yes